### PR TITLE
fix: Style updates per dark theme audit

### LINKED ISSUE #63: Updates per dark theme audit
**Describe the problem**
Results of dark theme audit:

- Dark Theme: subtitle color on select same as title for all status
- Light Theme: default and warning status subtitles same color as title
- Both: Edge lines should be status color (?)
- Both: Selected group hover border:
- Light theme: selected group border color --pf-global--primary-color--200
- Dark theme: selected group border color --pf-global--primary-color--light-100 

/cc @andrew-ronaldson @mceledonia

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## What
Fixes #63

## Description
Update color styles per [dark theme audit](https://github.com/patternfly/react-topology/issues/49).

## Type of change
- [x] Bugfix

### Light Theme Labels:
image

### Dark Theme Labels:
image

### Light Theme Selected Group Hover:
image

### Dark Theme Selected Group Hover:
image

/cc @mceledonia @andrew-ronaldson

### CODE REVIEW

**Comment by nicolethoen** [comment]
@mcoker and @mceledonia and @andrew-ronaldson could you all take a peek at this?

**Comment by jeff-phillips-18** [comment]
@mcoker and @mceledonia and @andrew-ronaldson could you all take a peek at this?

**Comment by jeff-phillips-18** [comment]
Rebased.

Surge: https://topology-dark-theme.surge.sh

File: packages/module/src/css/topology-components.css [hunk 1]
```diff
@@ -193,14 +199,20 @@
...
   --pf-topology__node--m-selected--node__label__background--Fill: var(--pf-v5-global--palette--blue-400);
-  --pf-topology__node--m-selected--separator--Stroke: var(--pf-v5-global--palette--blue-300);
 
   --pf-topology__node--m-selected--node__label__text--Fill: var(--pf-v5-global--Color--100);
   --pf-topology__node--m-selected--m-info--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
   --pf-topology__node--m-selected--m-danger--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
   --pf-topology__node--m-selected--m-warning--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
   --pf-topology__node--m-selected--m-success--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
 
+  --pf-topology__node--m-selected--node__label__text--m-secondary--Fill: var(--pf-v5-global--palette--black-100);
```
Comment (mcoker) [inline packages/module/src/css/topology-components.css:209]: The secondary label text color doesn't match the primary label, but it does match in the other label status colors and it does in light theme, too. If you wanted it to match, you could update this to `var(--pf-v5-global--Color--100);`. It doesn't look like your other vars are setup this way, but if you wanted the primary/secondary to always match, you could use `var(--pf-topology__node--m-selected--node__label__text--Fill)` here instead.

File: packages/module/src/css/topology-components.css [hunk 2]
```diff
@@ -631,17 +642,21 @@
 
 .pf-topology__edge.pf-m-info {
   --edge--stroke: var(--pf-topology__edge--m-info--EdgeStroke);
+  --edge--fill: var(var(--pf-v5-global--primary-color--light-100));
```
Comment (mcoker) [inline packages/module/src/css/topology-components.css:645]: typo

```suggestion
  --edge--fill: var(--pf-v5-global--primary-color--light-100);
```

**Review by mcoker — APPROVED** [review]
LGTM! Everything here (except for the double `var()` wrapping `--edge--fill: var(var(--pf-v5-global--primary-color--light-100));`) is a question/observation, not necessarily a suggestion. 

* `--pf-topology-pipelines__pill-badge--fill` and `--pf-topology-pipelines__pill-badge--text--fill` look like a read badge, and we use `var(--pf-v5-global--palette--black-500)` and `var(--pf-v5-global--Color--100)` as the background and text color in read badges. Here's what the badge looks like with that
  Screenshot 2023-06-20 at 6 51 37 PM
  * for reference here is what it looked like before
    Screenshot 2023-06-20 at 6 52 01 PM
* For selected nodes with labels, it's worth noting that the selected node outline/stroke and the label background/fill are different colors (they are not different in light theme). That may be intentional. Also the selected node label fill (`--pf-topology__node__label__background--[Fill/Stroke]` is using `var(--pf-v5-global--palette--blue-400)`. That's the right color, but we have a semantic var for that background color (used in buttons and other places) that would be better to use if you can -`var(--pf-v5-global--primary-color--300)`. FWIW, this is what the selected node looks like if the stroke matches the label fill
  Screenshot 2023-06-20 at 7 09 41 PM
  * for reference here is what it looked like before
    Screenshot 2023-06-20 at 7 09 26 PM
* Some of these label badges look like they may have contrast issues?
  Screenshot 2023-06-20 at 7 11 42 PM

**Comment by jeff-phillips-18** [comment]
Fixed the double var error.

> Some of these label badges look like they may have contrast issues?

These are custom colors used by the demo app just to validate setting custom colors, not part of the toolkit

@mceledonia @andrew-ronaldson Could you take a look at @mcoker's comments above and let me know if we should update as he suggests?

**Comment by jeff-phillips-18** [comment]
I have updated per @mcoker's suggestions.

https://topology-dark-theme.surge.sh has been updated. PTAL.

**Comment by mceledonia** [comment]
Those suggestions and the theme overall look great!

**Pull request merged**